src/app/workflow/workflow.component.ts

```diff
@@ -80,6 +82,10 @@ export class WorkflowComponent extends Entry implements AfterViewInit {
   public schema;
   public extendedWorkflow$: Observable<ExtendedWorkflow>;
   public WorkflowModel = Workflow;
+
+  public tokens: Token[];
```

Comment:
You don't need this any more.

Reply:
not applicable since code has been refactored according to Gary's suggestions